lightning/src/ln/channel.rs

```diff
@@ -53,19 +53,37 @@ enum InboundHTLCRemovalReason {
 }
 
 enum InboundHTLCState {
-	/// Added by remote, to be included in next local commitment tx.
+	/// Offered by remote, to be included in next local commitment tx. I.e., the remote sent an
+	// update_add_htlc message for this HTLC.
 	RemoteAnnounced(PendingHTLCStatus),
-	/// Included in a received commitment_signed message (implying we've revoke_and_ack'ed it), but
-	/// the remote side hasn't yet revoked their previous state, which we need them to do before we
-	/// accept this HTLC. Implies AwaitingRemoteRevoke.
-	/// We also have not yet included this HTLC in a commitment_signed message, and are waiting on
-	/// a remote revoke_and_ack on a previous state before we can do so.
+	/// Included in a received commitment_signed message (implying we've revoke_and_ack'd it), but
+	/// the remote hasn't yet revoked their previous state (see the example below). We have not yet
+	/// included this HTLC in a commitment_signed message because we are waiting on the remote's
+	/// aforementioned state revocation. The reason we need this remote RAA (revoke_and_ack) is
```

Comment:
I feel a bit awkward calling this *the* reason - is true, it is a reason, but the other reason is because RAA "commits" a state, but doesn't specify which state, leaving an implication of it ACK'ing the latest state (commitent_signed that we sent).

Reply:
Let me know if the new version is good. I added the second reason as the last sentence of the paragraph